README.md

```diff
@@ -103,3 +103,13 @@ $ npm link hydrogen-view-sdk
 ### Running tests
 
 See the [testing readme](./test/README.md).
+
+## Community
+
+Got any questions? Join our community chat :
+
+  <a href="https://matrix.to/#/#matrix-public-archive:matrix.org"><img src="https://img.shields.io/badge/chat-%23matrix--public--archive-success" alt="chat" /></a>
```

Comment:
Current suggested badge:

https://img.shields.io/badge/chat-%23matrix--public--archive-success
![](https://user-images.githubusercontent.com/558581/205730784-a3c59c61-058c-4428-bcb6-f7b157eeae43.png)

To compare with other badges:

https://img.shields.io/matrix/matrix-public-archive:matrix.org?label=support&logo=matrix (the one from the [Synapse readme](https://github.com/matrix-org/synapse/blob/6a8310f3dfe77acf59df2fe3e88a71b85b9b3ecc/README.rst))
![](https://user-images.githubusercontent.com/558581/205730667-0d92759f-273b-488b-81b1-7af051e8c711.png)

https://img.shields.io/matrix/matrix-public-archive:matrix.org.svg?label=%23matrix-public-archive%3Amatrix.org&logo=matrix&server_fqdn=matrix.org (the one from the [Dendrite readme](https://github.com/matrix-org/dendrite/blob/75834783055b0c70f8b411d9c3741e57461832f0/README.md))
![](https://user-images.githubusercontent.com/558581/205731737-90dd2b33-7a4b-43e3-b5f6-59ff610c6690.png)


---

The Dendrite one seems the best ⏩